task_06.py

```diff
@@ -11,3 +11,5 @@
 WORDS = FHANDLER.read()
 
 FHANDLER.close()
+
+WORDCT =  len(WORDS.split(' '))
```

Comment:
`WORDS.split()` -- your `split(' ')` only breaks on spaces but the default form breaks on all whitespace including newlines and tabs.